randovania/games/dread/json_data/Ghavoran.txt

```diff
@@ -691,10 +697,11 @@ Extra - asset_id: collision_camera_009
   * Extra - right_shield_def: actordef:actors/props/doorshieldmissile/charclasses/doorshieldmissile.bmsad
   > Bomb Ledge
       Any of the following:
-          Grapple Beam or Space Jump or Simple IBJ
+          Space Jump or Simple IBJ or Use Spin Boost
```

Comment:
Sure this isn´t trivial?

Reply:
Yes, you have to use spin boost to wall jump to the ledge

Reply:
`Use Spin Boost` covers having Space Jump.